Update luatex .ini files for compatibility with latest luatex versions

From TeX Live svn.

Index: work/texmf/tex/generic/config/luatexiniconfig.tex
===================================================================
--- /dev/null
+++ work/texmf/tex/generic/config/luatexiniconfig.tex
@@ -0,0 +1,11 @@
+% Enable LuaTeX primitives for INI mode (needed since 0.39.0)
+% Élie Roux & Manuel Pégourié-Gonnard, 2009. Public domain.
+\begingroup
+\catcode`\{=1
+\catcode`\}=2
+\expandafter\ifx\csname luatexversion\endcsname\relax
+  \let\luatexversion\undefined
+  \directlua{tex.enableprimitives('', tex.extraprimitives())}
+\fi
+\endgroup
+\endinput
Index: work/texmf-dist/tex/plain/config/luatex.ini
===================================================================
--- work.orig/texmf-dist/tex/plain/config/luatex.ini
+++ work/texmf-dist/tex/plain/config/luatex.ini
@@ -1,3 +1,4 @@
 % Karl Berry, 2008.  Public domain.
 % DVI-output plain luatex.
+\input luatexiniconfig.tex
 \input etex.ini
Index: work/texmf-dist/tex/plain/config/pdfluatex.ini
===================================================================
--- work.orig/texmf-dist/tex/plain/config/pdfluatex.ini
+++ work/texmf-dist/tex/plain/config/pdfluatex.ini
@@ -1,4 +1,5 @@
 % Karl Berry, 2008. Public domain.
+\input luatexiniconfig.tex
 \input pdftexconfig.tex
 \input plain
 \dump
